src/packages/@ncigdc/components/Layouts/MasonryLayout.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+
+interface IElementProps {
+  component: string | JSX.Element;
+  size: number;
+}
+
+interface IMasonryLayoutProps {
+  elements: IElementProps[];
+  numPerRow: number;
+  minWidth?: number;
+  gutter?: number;
+  customizeStyle?: React.CSSProperties;
+}
+
+
+const MasonryLayout = ({
+  customizeStyle = {
+    borderStyle: 'solid', // Replace it with your own style
```

Comment:
i think this comment means it's expecting a custom style prop?

Reply:
Pretty sure that's for whoever comes after (e.g. myself) to style the cards accordingly.

Reply:
Lol I just totally read this prop incorrectly 🤦‍♀️Please disregard!